templates/demo.html

```diff
@@ -7,6 +7,27 @@
 {% block content %}
 <script type="module">
   import AnboxStream from '/static/js/anbox-stream-sdk.js';
+
+  function isMobile() {
```

Comment:
@carkod did a similar refactor in https://github.com/canonical-web-and-design/anbox-cloud.io/pull/89